openhands/server/session/conversation.py

```diff
@@ -37,3 +37,6 @@ def __init__(
 
     async def connect(self):
         await self.runtime.connect()
+
+    async def disconnect(self):
+        self.runtime.close()
```

Comment:
**Suggestion:** Await the `disconnect` method as it is asynchronous.

**Existing Code:**
```Python
self.runtime.close()
```

**Improved Code:**
```Python
await self.runtime.close()
```

**Details:**
Ensure `disconnect` is awaited since it is an asynchronous function.